exercises/house/canonical-data.json

```diff
@@ -0,0 +1,322 @@
+{
+  "exercise": "house",
+  "version": "1.0.0",
+  "comments": [
+    "JSON doesn't allow for multi-line strings, so all verses are presented ",
+    "here as arrays of strings. It's up to the test generator to join the ",
+    "lines together with line breaks.",
+    "Some languages test for the verse() method, which takes a start verse ",
+    "and optional end verse, but other languages have only tested for the full poem.",
+    "For those languages in the latter category, you may wish to only ",
+    "implement the full song test and leave the rest alone, ignoring the start verse, ",
+    "end verse, and verse fields."
+  ],
+  "cases": [
+    {
+      "description": "Return specified verse or series of verses",
+      "cases": [
+        {
+          "description": "verse one",
+          "property": "verse",
+          "verse": 1,
+          "expected": [
+            "This is the house that Jack built."
+          ]
+        },
+        {
+          "description": "verse two",
```

Comment:
@Smarticles101 Are you interested in updating this PR to add those more descriptive descriptions?